### PR TITLE
Add `analytics/track-event` capability with mixpanel provider

### PR DESCRIPTION
**Pull request by freaz**
## Description

Adds profile to track an event in analytics systems such as Google analytics, Mixpanel, Crisp.chat, etc.

In PR only Mixpanel with EU Api is implemented.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by jnv — APPROVED** [review]
LGTM

File: capabilities/analytics/track-event/maps/mixpanel-eu.suma [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      body {
+        data = {
+          event: input.eventName,
+          properties: { token: input.token, ...input.eventProperties }
+        }
+      }
+    }
+  
+    response 200 "*" {
+      return map result if (body === 1) "OK"
```
Comment (zdne) [inline capabilities/analytics/track-event/maps/mixpanel-eu.suma:20]: haha that is ridiculous API

File: capabilities/analytics/track-event/maps/mixpanel-eu.test.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          provider: 'mixpanel-eu',
+          useCase: 'TrackEvent',
+          input: {
+            eventName: 'stationtest',
+            eventProperties: {
+              user: 'user_1',
+            },
+
+            // OneSDK doesn't support credentials in bodies (this must be replaced with actual key when running against live API)
+            token: 'xxx',
```
Comment (zdne) [inline capabilities/analytics/track-event/maps/mixpanel-eu.test.ts:24]: so this is an integration parameter ...

File: capabilities/analytics/track-event/profile.supr [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    """
+    Event Properties
+    Event Properties are attributes that help you define the specifics of an Event.
+    """
+    eventProperties
+  }
+
+  result string
+
+  error {
```
Comment (zdne) [inline capabilities/analytics/track-event/profile.supr:29]: Not for this PR but since this repeats so much I wonder if we can't make it somehow standard..

**Review by zdne — APPROVED** [review]
👍🏻

**Comment by freaz** [comment]
I will open new PR with provider with integration parameters.

**Pull request closed without merging**